Configuration/Geometry/python/dict2023Geometry.py

```diff
@@ -282,11 +282,11 @@
             'Geometry/HcalSimData/data/CaloUtil.xml',
             'Geometry/HGCalSimData/data/hgcsensv8.xml',
             'Geometry/HGCalSimData/data/hgccons.xml',
-            'Geometry/HGCalSimData/data/hgcProdCuts.xml',
         ],
         4 : [
             'Geometry/HcalSimData/data/HcalProdCuts.xml',
             'Geometry/EcalSimData/data/EcalProdCuts.xml',
+            'Geometry/HGCalSimData/data/hgcProdCuts.xml',

```

Comment:
if you make this change, you have to rerun the geometry script for all geometries that contain C3.